python/ray/tune/schedulers/pbt.py

```diff
@@ -47,7 +47,13 @@ def explore(config, mutations, resample_probability, custom_explore_fn):
     """
     new_config = copy.deepcopy(config)
     for key, distribution in mutations.items():
-        if isinstance(distribution, list):
+        if isinstance(distribution, dict):
+            new_config.update({
+                key: explore(
+                    config[key], mutations[key],
+                    resample_probability, custom_explore_fn)
```

Comment:
Should we pass a no-op custom_explore_fn here? It might be weird to be invoking that at multiple levels.

Reply:
Makes sense not to invoke the custom preturbation function at every level. But instead of passing a no-op, I believe we should pass the `custom_explore_fn` as is but only invoke it at the last level? What do you think?